DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ConferenceCell.java

```diff
@@ -46,8 +45,8 @@
 import javafx.scene.layout.VBox;
 import javafx.scene.shape.Rectangle;
 
-import java.time.LocalDate;
-import java.time.format.DateTimeFormatter;
+//import java.time.LocalDate;
```

Comment:
if possible, switch back to use java.time.format.DateTimeFormatter otherwise add TODOs so we don't forget later